semantic-python/src/Language/Python/Core.hs

```diff
@@ -0,0 +1,141 @@
+{-# LANGUAGE DefaultSignatures, DeriveGeneric, FlexibleContexts, FlexibleInstances, RecordWildCards, StandaloneDeriving, TypeOperators #-}
+module Language.Python.Core
+( compile
+) where
+
+import           Control.Monad.Fail
+import           Data.Core as Core
+import           Data.Name as Name
+import           GHC.Generics
+import           Prelude hiding (fail)
+import qualified TreeSitter.Python.AST as Py
+
+class Compile t where
```

Comment:
While the implementation of this file is by no means complete, it’s adequate to spike out _a_ workflow. Further refinement of this, including the development of a Core representation of any necessary stdlib components, will be taken up in future PRs.